tests/semanticdb/metac.expect

```diff
@@ -3469,10 +3624,10 @@ Symbols => 4 entries
 Occurrences => 5 entries
 
 Symbols:
-exports/`exports-package$package`. => final package object exports
-exports/`exports-package$package`.Codec# => final type Codec
-exports/`exports-package$package`.Decoder# => final type Decoder
-exports/`exports-package$package`.Encoder# => final type Encoder
+exports/`exports-package$package`. => final package object exports extends Object { self: exports.type => +4 decls }
+exports/`exports-package$package`.Codec# => final type Codec  = Codec[<?>]
+exports/`exports-package$package`.Decoder# => final type Decoder  = Decoder[<?>]
+exports/`exports-package$package`.Encoder# => final type Encoder  = Encoder[<?>]
```

Comment:
For the export clause

```scala
export example.{Decoder, Encoder, Codec}
```

I found there's type argument `TypeParamRef("T")` for `Codec`, `Decoder`, and `Encoder`, whose binder is

```scala
HKTypeLambda(
  List(T),
  List(TypeBounds(TypeRef(ThisType(TypeRef(NoPrefix,module class scala)),class Nothing),TypeRef(ThisType(TypeRef(NoPrefix,module class scala)),class Any))), 
  AppliedType(
    TypeRef(TermRef(ThisType(TypeRef(NoPrefix,module class exports)),object example),trait Decoder),
    List(TypeParamRef(T))
  )
)
```

where this `HKTypeLambda` never appears in the source level (which means we can't register this typeparamref to the symbol table).

Reply:
Currently, I suppress the warning of symbol lookup failure from this situation https://github.com/lampepfl/dotty/pull/12885/commits/d14cfa5b210ea3572eb6986c9d7b020e3b2f6309 because it's a known issue and I'm not sure how can we fix 🤔 
I guess we can fix this by creating a symbol table across multiple TextDocuments, but I think it cost a ton of memory for a big project.

Reply:
This is also the problem of `TypeTree`, maybe we can work-around this by creating new symbol.

Reply:
It also works with `Type.member` https://github.com/lampepfl/dotty/pull/12885/commits/5913b023bd13ee84e03f78e8aee335295e088aef